llvm/lib/Target/DPU/DPUTargetLowering.cpp

```diff
@@ -1810,10 +1810,27 @@ SDValue DPUTargetLowering::LowerDMA(SDValue Op, SelectionDAG &DAG,
 }
 
 static uint64_t PageSizeLog2ToNcCondition(uint64_t pageSize) {
-  if (pageSize >= 10) {
-    return DPUAsmCondition::NotCarry10 + pageSize - 10;
-  } else {
-    return DPUAsmCondition::NotCarry5 + pageSize - 5;
+  switch (pageSize) {
```

Comment:
No default case? Does the compiler warn you about it?

Reply:
Fixed